packages/uui-popover-container/lib/uui-popover-container.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Primary, Controls, Story } from '@storybook/blocks';
+import * as stories from './uui-popover-container.story';
+
+<Meta of={stories} />
```

Comment:
We should make this one appear under "Displays". And then we should be even more explicit about that it does not work in Firefox. So the devs in Cloud Portal should not go ahead and implement this now.